app/src/main/cpp/skyline/services/btm/IBtmUserCore.h

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2022 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <services/serviceman.h>
+
+namespace skyline::service::btm {
+    /**
+     * @brief IBcatService is used to interact BLE (Bluetooth Low Energy) devices
+     * @url https://switchbrew.org/wiki/BTM_services#IBtmUserCore
+     */
+    class IBtmUserCore : public BaseService {
+      public:
+        IBtmUserCore(const DeviceState &state, ServiceManager &manager);
+
+        /**
+         * @brief No input, returns an output bool and an Event handle with EventClearMode=1.
```

Comment:
These comments can be dropped apart from URL, we don't do them anymore

Reply:
Removed.